src/Parser/Core/Modules/Items/Legion/AntorusTheBurningThrone/EyeOfHounds.js

```diff
@@ -0,0 +1,139 @@
+import React from 'react';
+
+import ITEMS from 'common/ITEMS';
+import SPELLS from 'common/SPELLS';
+import ItemLink from 'common/ItemLink';
+import Analyzer from 'Parser/Core/Analyzer';
+import Wrapper from 'common/Wrapper';
+import Combatants from 'Parser/Core/Modules/Combatants';
+import { formatPercentage } from 'common/format';
+import { calculatePrimaryStat, calculateSecondaryStatDefault } from 'common/stats';
+import Abilities from 'Parser/Core/Modules/Abilities';
+
+/**
+ * Eye of Shatug / Eye of F'harg
+ * Use: swaps between both versions
+ * initial state is in the trinkets bonusID (3618 for the stamina one, 3619 for the armor one)
+*/
+
+const ARMOR_EYE_BONUS_ID = 3619;
+
+class EyeOfHounds extends Analyzer {
+  static dependencies = {
+    combatants: Combatants,
+    abilities: Abilities,
+  };
+
+  isCurrentlyVers = true;
+  lastCheck = 0;
+  
+  stamStat = 0;
+  versStat = 0;
+
+  strStat = 0;
+  armorStat = 0;
+
+
+  stamUptime = 0;
+  strUptime = 0;
+
+  on_initialized() {
+    this.active = this.combatants.selected.hasTrinket(ITEMS.EYE_OF_HOUNDS.id);
+    if (this.active) {
+      const iLvl = this.combatants.selected.getItem(ITEMS.EYE_OF_HOUNDS.id).itemLevel;
+      const bonusIDs = Object.values(this.combatants.selected.gear).filter(item => item.id === ITEMS.EYE_OF_HOUNDS.id)[0].bonusIDs;
+      
+      if (bonusIDs.includes(ARMOR_EYE_BONUS_ID)) {
+        this.isCurrentlyVers = false;
```

Comment:
You could shorten this to `this.isCurrentlyVers = !bonusIDs.includes(ARMOR_EYE_BONUS_ID);`. And if you want it a bit prettier without the invert, you could check for the vers bonus ID instead.